GITOID_URI_SPEC.md

```diff
@@ -0,0 +1,44 @@
+Scheme name: gitoid
+
+Status: provisional
+
+Applications/protocols that use this scheme name:
+     See [Section 3.5](https://www.rfc-editor.org/rfc/rfc7595.html#section-3.5).
+Contact:
+     Registering party: Ed Warnicke <eaw@cisco.com>
+
+Scheme Creator: GitBOM
+
+Change controller:
+    Either the registering party or someone who is verified to represent
```

Comment:
Should reword to: "Change controller: Either the registering party or someone verified to represent the scheme creator. See the previous answer."

Reply:
I believe I've fixed this.